resources/adobe-experience-platform-mini-programs-sdk/faq.md

```diff
@@ -22,8 +22,8 @@ Ensure that your SDK timestamp configuration is aligned with the report suite's
 
 The following settings explain how settings between the SDK and your report suite should be aligned:
 
-* `analytics.offlineEnabled = true` ties to timestamps required or optional
-* `analytics.offlineEnabled = false` ties to timestamps not allowed or optional       
+* `analytics.offlineEnabled = true` ties to timestamp-required or timestamp-optional report suites
+* `analytics.offlineEnabled = false` ties to timestamp-forbidden or timestamp-optional report suites
```

Comment:
we should use not allowed here, not forbidden, this is the name from Analytics https://experienceleague.adobe.com/docs/analytics/admin/admin-tools/timestamp-optional.html?lang=en#section_095A7CFBD280494593B9BEC1592B73A6

Reply:
Ahh ok, got it. I'll re-phrase accordingly.